src/mixins/filing-template-mixin.ts

```diff
@@ -389,7 +391,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     this.setFilingId(+draftFiling.header.filingId)
 
     // restore Business Address
-    this.setRegistrationBusinessAddress(draftFiling.registration.businessAddress)
+    this.setRegistrationBusinessAddress(draftFiling.registration.offices?.businessOffice)
```

Comment:
Why use conditional chaining (`?`) here? In what scenarios could the offices object be undefined in the draft filing?

Reply:
When we receive initial draft from auth-web the office object won't be available.

Reply:
I see. So this code will assign `undefined` to the Registration Business Address object. Have you tested that the code (step 1) handles this fine?

Reply:
Yes it did work